pandas/core/indexing.py

```diff
@@ -1920,11 +1920,35 @@ def _convert_key(self, key, is_setter=False):
 
 
 class _iAtIndexer(_ScalarAccessIndexer):
-    """Fast integer location scalar accessor.
+    """
+    Selects a single value for a row/column pair by integer position.
```

Comment:
Selects -> Select

Reply:
BTW, I think "Select" is not the correct term, as you can also use this to set values